app/assets/stylesheets/common/components/user-card.scss

```diff
@@ -112,232 +126,606 @@
+  }
+  &__header {
+    position: relative;
+    height: 100px;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: space-between;
+  }
+  &__badges {
+    display: flex;
+    border-radius: var(--d-border-radius);
+    gap: 0.5em;
+    margin-left: auto;
+  }
+  &__badges .user-badge {
+    background-color: transparent;
+    border: none;
+    margin-top: 0;
+    padding: 0;
+  }
+  &__badges .user-card-badge-link {
+    display: flex;
+    padding: 5px 6px;
+    background-color: rgba(var(--secondary-rgb), 1);
+    border-radius: var(--d-border-radius);
+  }
+  &__badges .user-card-badge-link:hover {
+    background-color: var(--secondary-very-high);
+  }
+  &__badges .badge-display-name {
+    display: none;
+  }
+  &__badges .svg-icon {
+    width: 1.125em;
+    height: 1.125em;
+  }
+  &__badges-more {
+    font-size: var(--font-down-1);
+    display: flex;
+    padding: 5px 6px;
+    background-color: rgba(var(--secondary-rgb), 1);
+    border-radius: var(--d-border-radius);
+    align-items: center;
+  }
+  &__badges-more:hover {
+    background-color: var(--secondary-very-high);
+  }
+  &__badges-more a {
+    font-size: var;
+    color: var(--primary);
+  }
+  &__edit-btn {
+    cursor: pointer;
+    position: absolute;
+    right: 0;
+    bottom: 5px;
+    border: none;
+    font-size: 14px;
+    font-weight: normal;
+    transition: all 0.25s;
+    opacity: 50%;
+    color: var(--primary) !important;
+    background-color: rgba(var(--secondary-rgb), 0.75) !important;
+    &:hover {
+      opacity: 100%;
     }
-    a {
-      color: var(--primary);
-      text-decoration: underline;
+  }
+  &__badge .fa-crab {
+    color: var(--danger);
+  }
+  &__badge .fa-fish {
+    color: var(--success);
+  }
+  &__badge .fa-bird {
+    --fa-primary-color: var(--tertiary);
+    --fa-secondary-color: var(--highlight-high);
+  }
+  &__badge .fa-bowl-scoop {
+    --fa-primary-color: var(--danger-low);
+    --fa-secondary-color: var(--love);
+    --fa-secondary-opacity: 1;
+  }
+  &__main-content {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    border-radius: var(--d-border-radius);
+    background-color: var(--secondary);
+    padding: 0.5em;
+  }
+  &__main-content-top {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-content: center;
+    flex: 1 1 min-content;
+  }
+  &__main-content-bottom {
+    flex: 1 1 auto;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: flex-end;
+    gap: 1em;
+    margin: 0.75em 0 0 0;
+  }
+  &__id {
+    display: flex;
+    min-height: 50px;
+    position: relative;
+    max-width: calc(480px - 1em);
+    flex-wrap: wrap;
+    height: 100%;
+  }
+  &__id .d-user-card__avatar + div:has(> .avatar-flair) {
+    position: absolute;
+    left: 100%;
+  }
+  &__id .avatar-flair {
+    width: 40px;
+    height: 40px;
```

Comment:
Maybe move the avatar width here into a variable and reuse it? I'm guessing that multiple other pixel values are interrelated, it might be nice to use variables to indicate dependencies in this file.